csrc/scheduler/vectorize_helper.cpp

```diff
@@ -841,6 +841,84 @@ std::vector<std::unordered_map<TensorView*, Val*>> getTvToContigInnerSizeMapsOf(
   return mappers;
 }
 
+// The returns a map of input to their resize alignment info. This function
+// currently checks the alignment requirement on contiguous iterdomain in
+// allocation domain that *could be* broken by resize. This function returns a
+// map for TensorView, containing the index in their allocation domain whose
+// stride needs to be checked in alignment. e.g. For TensorView with size {4,
+// 1025}, assuming it's contiguous on all dimensions. If we slice the second
+// dimension with slice(0, 1024, 1). The output tensor would have shape {4,
+// 1024}. The slice prevents us from collapsing the indexing on its outer
+// dimension. At this point, due to the resize operation on dimension-1024, we
+// now need to check the stride on dimension-4 for alignment. NOTE: This is a
+// conservative check, not all resize breaks the contiguity of its inner
+// dimension, i.e. think about a combination of pad/slice that cancels out each
+// other;
```

Comment:
now that we further restricted how vectorization width is propagated across resize, I'm not sure whether we still need this check.

The example mentioned above won't be vectorized any more, because we are taking GCD on each resize extent. given `t0[4, 1025] -> slice(:, :1024) -> t1[4, 1024]`, the resize extent is just 1 so we won't vectorize.

So that means, if the projected extent after resize is vectorizable, I think it's safe to say the `stride` on the immediate outer dimension of the inner-most resize should always be compatible with the vectorizable width.

This means I can't really get a test to validate this PR, even though I still think it's a good to have thing, in case we improve the propagation.

I'm open to suggestions.